suricata.yaml.in

```diff
@@ -1685,6 +1685,10 @@ detect:
     toclient-groups: 3
     toserver-groups: 25
   sgh-mpm-context: auto
+  # Cache MPM contexts to the disk for avoid rule compilation at the startup.
```

Comment:
interestingly, [this version](https://github.com/OISF/suricata/pull/11708/files#diff-6edad567c7414ca3b78009e7d315547542cbb77455e7a223c11719d2734fd3dbR1675) had better preposition usage :P